packages/ui/src/components/organisms/ScrollList/ScrollList.tsx

```diff
@@ -48,6 +59,8 @@ const ScrollList = ({
   translator,
   horizontal = false,
   disabledRefresh = false,
+  actionList,
+  verticalAction,
```

Comment:
when vertical Action is false this is displayed in vertical setting, maybe horizontal  is better ?